src/Scope.ts

```diff
@@ -834,21 +834,25 @@ export class Scope {
                             : new NamespaceType(nameSoFar);
                         if (previousNSType) {
                             // adding as a member of existing NS
-                            previousNSType.addMember(nsNamePart, namespace.range, currentNSType, getTypeOptions.flags);
+                            previousNSType.addMember(nsNamePart, { definingNode: namespace }, currentNSType, getTypeOptions.flags);
```

Comment:
What was the need for the `definingNode` property? I'm not a huge fan of linking AST to types, would be nice to find a way to avoid this if possible.